apps/fluent-tester/src/E2E/common/NativeTesting/pages/NativeTestingPageObject.win.ts

```diff
@@ -0,0 +1,45 @@
+import { BasePage, By } from '../../../common/BasePage.win';
+
+class NativeTestingPageObject extends BasePage {
+  waitForScrollViewDisplayed(timeout?: number): void {
+    browser.waitUntil(
+      () => {
+        return this.doesScrollViewParentExist();
+      },
+      {
+        timeout: timeout ?? this.waitForPageTimeout,
+        timeoutMsg:
+          'The ScrollView containing all test page buttons was not found. Did you remove the ScrollView or change the testID? The testID must stay constant to align with our native code.',
+        interval: 1000,
+      },
+    );
+  }
+
+  doesScrollViewParentExist(): boolean {
+    return this._scrollViewParent.isExisting();
+  }
+
+  validateScrollViewChildren() {
+    // Gets all the children
+    const testChildren = this._scrollViewParent.$$('//*');
+
+    // All the button children have the same format. We want to find these buttons
```

Comment:
nit would tweak to something like // Ensure the AutomationId properties of the button children match the defined testing format

Reply:
Changing in newest commit